### PR TITLE
Fix #13 : doesn't throw error when listing the installed version if t…

### LINKED ISSUE #13: `psvm ls` fails if the psvm prefix directory does not exist
To reproduce:

```
$ rm -r $HOME/.psvm
$ psvm ls
s.js:945
  return binding.readdir(pathModule._makeLong(path), options.encoding);
                 ^

Error: ENOENT: no such file or directory, scandir '/Users/hgarrood/.psvm/versions'
    at Error (native)
    at Object.fs.readdirSync (fs.js:945:18)
    at Object.getInstalledVersions (/Users/hgarrood/.npm-packages/lib/node_modules/psvm/src/lib.js:30:17)
    at Object.ls [as action] (/Users/hgarrood/.npm-packages/lib/node_modules/psvm/src/cli.js:20:22)
    at Object.cli.execute (/Users/hgarrood/.npm-packages/lib/node_modules/psvm/node_modules/cliparse/src/cli.js:80:30)
    at Object.cli.parse (/Users/hgarrood/.npm-packages/lib/node_modules/psvm/node_modules/cliparse/src/cli.js:153:9)
    at Object.<anonymous> (/Users/hgarrood/.npm-packages/lib/node_modules/psvm/src/cli.js:139:10)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:4
```

I guess ideally it would simply output nothing and exit with 0 in this case.

### PR DESCRIPTION
**Pull request by ThomasCrevoisier**
…he .psvm folder doesn't exist

### CODE REVIEW

**Comment by ThomasCrevoisier** [comment]
@hdgarrood I let review this PR, let me know if you have comments :)

File: src/lib.js [hunk 1]
```diff
@@ -3,130 +3,137 @@ var got = require('got'),
...
-    return R.match(/v?\d+\.\d+\.\d+/, dir).length > 0;
-  }, dirs);
+function getInstalledVersions() {
+    return fs.readdirAsync(paths.PSVM_VERSIONS)
+        .then(function (dirs) {
+            return R.filter(function (dir) {
+                return R.match(/v?\d+\.\d+\.\d+/, dir).length > 0;
+            }, dirs);
+        }, function (err) {
+			if (err.code === 'ENOENT') {
```
Comment (hdgarrood) [inline src/lib.js]: Indentation looks a bit off here

**Comment by hdgarrood** [comment]
Looks good. Usually I prefer to keep large whitespace changes in separate commits for easy reviewing and a more useful history, but since you've done the work already, I don't think it's worth going back and changing it now.

File: src/lib.js (hunk 1 above)
Comment (ThomasCrevoisier) [inline src/lib.js]: Could you share the `.editorconfig` you use ? Or the formatting convention you have ?

Comment (hdgarrood) [inline src/lib.js]: I don't use anything like that. I just go by what the surrounding code looks like - in the rest of this file, it looks like you're using 4 spaces everywhere, but here, it looks like you're using tabs.

**Comment by ThomasCrevoisier** [comment]
@hdgarrood What are you referring by `whitespace changes` ? About the formatting ?

**Comment by hdgarrood** [comment]
Yeah, I just mean things that don't change what the code actually does, but just modify indentation, or move braces on to the next or previous line, or add or remove spaces before the `()` in a function declaration, and so on. Most of the changes in src/lib.js look like whitespace changes to me.

**Comment by ThomasCrevoisier** [comment]
Yep, my bad I reformatted the whole file by reflex :/

**Comment by ThomasCrevoisier** [comment]
I reworked the commit to only keep the implementation changes

**Comment by hdgarrood** [comment]
👍 looks great

**Pull request merged**